service/src/main/kotlin/fi/espoo/evaka/reports/RawReport.kt

```diff
@@ -197,6 +203,112 @@ private fun Database.Read.getRawRows(from: LocalDate, to: LocalDate): List<RawRe
         .toList()
 }
 
+private fun Database.Read.getRawRowsWithNewServiceNeeds(from: LocalDate, to: LocalDate): List<RawReportRow> {
+    // language=sql
+    val sql =
+        """
+SELECT
+    t::date as day,
+    p.id as child_id,
+    p.date_of_birth,
+    date_part('year', p.date_of_birth) as birth_year,
+    date_part('year', age(t::date, p.date_of_birth)) as age,
+    p.language,
+    p.post_office,
+
+    pl.type as placement_type,
+
+    pl.unit_id,
+    u.name as unit_name,
+    ca.name AS care_area,
+    u.type as unit_type,
+    u.provider_type as unit_provider_type,
+
+    dgp.daycare_group_id,
+    dg.name as group_name,
+    dc.amount as caretakers_planned,
+    sa.count as caretakers_realized,
+
+    bc.unit_id as backup_unit_id,
+    bc.group_id as backup_group_id,
+
+    sn IS NOT NULL AS has_service_need,
+    sno.part_day,
+    sno.part_week,
+    sn.shift_care,
+    sno.daycare_hours_per_week AS hours_per_week,
+
+    an IS NOT NULL as has_assistance_need,
+    coalesce(an.capacity_factor, 1.0) as capacity_factor,
+    coalesce(capacity_factor, 1.0) * (CASE
+        WHEN u.type && array['FAMILY', 'GROUP_FAMILY']::care_types[] THEN 1.75
```

Comment:
Is this the same "1.75" as in the code above? If so, should this be a constant?

Reply:
Yeah, it probably should.